src/ci/github-actions/ci.yml

```diff
@@ -153,6 +153,10 @@ x--expand-yaml-anchors--remove:
         run: src/ci/scripts/install-clang.sh
         <<: *step
 
+      - name: select Xcode version
+        run: sudo xcode-select -s $SELECT_XCODE
+        if: success() && !env.SKIP_JOB && env.SELECT_XCODE
```

Comment:
When we switched to GHA we migrated all the setup code to standalone bash scripts in `src/ci/scripts/{name}.sh`. Could you move both the command and the `SELECT_XCODE`  condition to it? A comment on top of the script to explain why that's needed would also be great!

Reply:
Instead, I just merged it with `install-clang` since the two were so intertwined anyway. That also removes the ordering issue.